packages/zipkin-instrumentation-express/src/expressMiddleware.js

```diff
@@ -79,7 +79,7 @@ module.exports = function expressMiddleware({tracer, serviceName = 'unknown', po
       const id = tracer.id;
 
       tracer.recordServiceName(serviceName);
-      tracer.recordRpc(req.method);
+      tracer.recordRpc(req.method.toUpperCase());
```

Comment:
All the other zipkin-js libraries are doing it? Should we remove this across them? 
Examples:
https://github.com/openzipkin/zipkin-js/blob/master/packages/zipkin-instrumentation-fetch/src/wrapFetch.js#L27

https://github.com/openzipkin/zipkin-js/blob/master/packages/zipkin-instrumentation-cujojs-rest/src/restInterceptor.js#L37